src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.component.html

```diff
@@ -1,109 +1,86 @@
 <div class="w-75 content-container">
     <h2>Compaction</h2>
     <mat-divider class="mt-1 mb-2"></mat-divider>
+    <p class="text-muted my-2">Compaction allows to merge huge amounts of dataset slices into one slice.</p>
 
-    <div class="box p-4 mt-3">
+    <div class="box p-4 mt-4">
         <h4 class="subtitle">Hard compaction</h4>
         <form [formGroup]="hardCompactionForm">
-            <mat-radio-group
-                class="d-flex flex-column row-gap-4"
-                formControlName="mode"
-                aria-label="Select an option"
-                color="primary"
-            >
-                <mat-radio-button [value]="CompactionMode.FULL">
-                    <span class="d-block">Full mode</span>
-                </mat-radio-button>
-                <div class="box p-4 box-margin-left">
-                    <div class="row">
-                        <div class="col-md-3">
-                            <div class="d-flex align-items-center pt-1">
-                                <app-tooltip-icon class="mt-1" [tooltip]="MAX_SLICE_SIZE_TOOLTIP"></app-tooltip-icon>
-                                <span class="d-inline-block">Max slice size:</span>
-                            </div>
+            <div class="box p-4 box-margin-left">
+                <div class="row">
+                    <div class="col-md-3">
+                        <div class="d-flex align-items-center pt-1">
+                            <app-tooltip-icon class="mt-1" [tooltip]="MAX_SLICE_SIZE_TOOLTIP"></app-tooltip-icon>
+                            <span class="d-inline-block">Max slice size:</span>
                         </div>
-                        <div class="col-md-9 d-flex">
-                            <select class="form-select me-4 form-control-width" formControlName="sliceUnit">
-                                <option [value]="SliceUnit.KB">KB</option>
-                                <option [value]="SliceUnit.MB">MB</option>
-                                <option [value]="SliceUnit.GB">GB</option>
-                            </select>
-                            <input
-                                type="number"
-                                min="1"
-                                class="form-control form-control-width"
-                                formControlName="sliceSize"
-                                data-test-id="slice-size"
-                            />
-                            <ng-container
-                                *ngIf="
-                                    sliceSizeControl.errors?.minNumber &&
-                                    (sliceSizeControl.touched || sliceSizeControl.dirty)
-                                "
+                    </div>
+                    <div class="col-md-9 d-flex">
+                        <select class="form-select me-4 form-control-width" formControlName="sliceUnit">
```

Comment:
Nitpick: Can we please switch numeric input and size units selector around so it was `300 MB` and not `MB 300`

Reply:
Fixed